library/modules/Items.cpp

```diff
@@ -866,38 +703,35 @@ static string get_item_type_str(df::item *item) {
 }
 
 static string get_base_desc(df::item *item) {
-    if (auto name = Items::getBookTitle(item); name.size())
+    if (auto name = Items::getBookTitle(item); !name.empty())
         return name;
-    if (auto artifact = get_artifact(item)) {
-        if (artifact->name.has_name)
-            return Translation::TranslateName(&artifact->name, false) + ", " + Translation::TranslateName(&artifact->name) + " (" + get_item_type_str(item) + ")";
-    }
+    if (auto artifact = get_artifact(item); artifact->name.has_name)
```

Comment:
this will crash if `get_artifact(item)` returns NULL. to combine the two if statements, it would have to be
```cpp
if (auto artifact = get_artifact(item); artifact && artifact->name.has_name)
```